src/routes/userRoutes.js

```diff
@@ -92,14 +92,14 @@ routes.loginUser = async (req, res) => {
   }
 }
   routes.updateDBPassword = async(req,res) =>{
-    if(!req.params.id ||!req.body.password ){
+    if(!req.session.username ||!req.body.password ){
```

Comment:
This is being a little nitpicky but try doing:

```
const username = req.session.username
const password = req.body.password
```

Reply:
Done!